test_schemas/en/test_new_routing_number_greater_than_or_equal_single_condition.json

```diff
@@ -0,0 +1,133 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "language": "en",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.3",
+    "survey_id": "001",
+    "title": "Test Routing Number Greater Than Or Equal To",
+    "theme": "default",
+    "description": "A test survey for routing based on a number greater than or equal to",
+    "metadata": [
+        {
+            "name": "user_id",
+            "type": "string"
+        },
+        {
+            "name": "period_id",
+            "type": "string"
+        },
+        {
+            "name": "ru_name",
+            "type": "string"
+        }
+    ],
+    "questionnaire_flow": {
+        "type": "Linear",
+        "options": {
+            "summary": {
+                "collapsible": false
+            }
+        }
+    },
+    "sections": [
+        {
+            "id": "default-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "number-question",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "answer",
+                                        "mandatory": true,
+                                        "type": "Number",
+                                        "label": "123 or greater"
+                                    }
+                                ],
+                                "id": "question",
+                                "title": "Enter the number greater than or equal to 123",
+                                "type": "General"
+                            },
+                            "routing_rules": [
+                                {
+                                    "goto": {
+                                        "block": "correct-answer",
+                                        "when": [
+                                            {
+                                                "id": "answer",
+                                                "condition": "greater than or equal to",
+                                                "value": 123
+                                            }
+                                        ]
+                                    }
```

Comment:
This is the old routing structure, the new operator structure need to be used so here `>=`.